packages/analytics-js/src/components/eventManager/utilities.ts

```diff
@@ -228,6 +228,15 @@ const getEnrichedEvent = (
   }
 
   const processedEvent = mergeDeepRight(rudderEvent, commonEventData) as RudderEvent;
+  // Set the default values for the event properties
+  // matching with v1.1 payload
+  if (processedEvent.event === undefined) {
```

Comment:
where does this take values from? it is custom ones provided by event method data?

Reply:
Yes, both `event` and `properties` are set in the respective factory methods (partial event object) if set by the user.